src/frequenz/channels/_merge.py

```diff
@@ -1,7 +1,51 @@
 # License: MIT
 # Copyright © 2022 Frequenz Energy-as-a-Service GmbH
 
-"""Merge messages coming from channels into a single stream."""
+"""Merge messages coming from multiple receivers into a single receiver.
+
+# Usage
+
+If you just need to receive the same type of messages but from multiple sources in one
+stream, you can use [`merge()`][frequenz.channels.merge] to create a new receiver that
+will receive messages from all the given receivers:
+
+```python
+from frequenz.channels import Anycast, Receiver, merge
+
+channel1: Anycast[int] = Anycast(name="channel1")
+channel2: Anycast[int] = Anycast(name="channel2")
+receiver1 = channel1.new_receiver()
+receiver2 = channel2.new_receiver()
+
+async for value in merge(receiver1, receiver2):
+    print(value)
+```
+
+If the first message comes from `channel2` and the second message from `channel1`, the
+first message will be received immediately, and the second message will be received as
+soon as it is available.
+
+This is extremely convenient when you just need to receive messages and don't care about
```

Comment:
Yeah, true, just "convenient" would do too. :smile:

Reply:
I'm going with "This can be convenient when..."

Reply:
Another possibility: "This can be helpful when ...".

Also might be worth mentioning: "... when you don't care about where they are coming from, or if the messages themselves have an id field that would help identify their origin or context ..."

Reply:
I find the last a bit too specific of a use case, and doesn't fit very nice to the following part talking about select, I would need to rephrase the whole thing only to include this very specific example, so I will keep that as it is for now. If you feel it is worth mentioning maybe you can provide the full rephrased paragraph?

Reply:
No, you're right.  It is probably already obvious that we're only talking about whether identity is available from the infrastructure.